docs/guide/android-dev-env.md

```diff
@@ -78,7 +78,7 @@ _Here’s a visual comparison between the two - an SDK 28 (Android 9) AOSP emula
 
 While it’s possible to do this using Android Studio, we’ll focus on the command line, as it is also good for _headless_ CI machines.
 
-1. Locate your 'Android home' folder - typically set in the `ANDROID_HOME` environment variable, or in its successor - `ANDROID_SDK_ROOT`. If `ANDROID_HOME` isn’t set, either set it yourself or run the following commands after `cd`-ing into the home folder.
+1. Locate your 'Android home' folder - typically set in the `ANDROID_HOME` environment variable, or in its predecessor - `ANDROID_SDK_ROOT`. If `ANDROID_HOME` isn’t set, either set it yourself or run the following commands after `cd`-ing into the home folder.
```

Comment:
Right, nice catch!

Reply:
Oh wow. At the time writing this, I was genuinely under the impression that `ANDROID_HOME` was the one they've deprecated. Thanks again!